cli/src/commands/init.rs

```diff
@@ -0,0 +1,51 @@
+//! Subcommand `phylum init`.
+
+use std::io;
+
+use clap::ArgMatches;
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::{Confirm, Input};
+
+use crate::api::PhylumApi;
+use crate::commands::{project, CommandResult};
+
+/// Handle `phylum init` subcommand.
+pub async fn handle_init(api: &mut PhylumApi, matches: &ArgMatches) -> CommandResult {
+    let cli_project = matches.get_one::<String>("project");
+    let cli_group = matches.get_one::<String>("group");
+
+    let (project, group) = match cli_project {
+        Some(project) => (project.clone(), cli_group.cloned()),
+        None => {
+            let project = prompt_project()?;
+            let group = match cli_group {
+                Some(group) => Some(group.clone()),
+                None => prompt_group()?,
+            };
```

Comment:
I feel like this part could also be made a bit clearer by chaining combinators, or switching to `if let`.